mulighetsrommet-api/src/main/resources/db/migration/V98__add_apent_for_innsok.sql

```diff
@@ -0,0 +1,8 @@
+alter table tiltaksgjennomforing
+    add apent_for_innsok boolean;
+
+update tiltaksgjennomforing
+set apent_for_innsok = tilgjengelighet != 'STENGT';
```

Comment:
Dette blir egentlig ikke helt riktig i forhold til hva som faktisk er satt i Arena siden vi ender opp med å sette `apent_for_innsok = false` for mange flere rader enn de som har `STATUS_TREVERDIKODE_INNSOKNING = N`.
Det vil føre til at vi viser flagget feil i adminflagte, men det burde føre til riktig oppførsel i veilederflate (der man benytter det i filteret).

Kjører en gjenspilling av alle gjennomføringer etter prodsetting slik at vi kommer i sync med riktig verdi igjen.